src/pages/Staking/index.tsx

```diff
@@ -103,7 +104,6 @@ interface LockingAmountAndTime {
 
 const Staking = (): JSX.Element => {
   const [blockLockTimeExtension, setBlockLockTimeExtension] = React.useState<number>(0);
-  const [availableBalance, setAvailableBalance] = React.useState(newMonetaryAmount(0, GOVERNANCE_TOKEN, true));
```

Comment:
@nud3l 
According to some best practices, we should not have zero configured as a default value because we cannot be sure that it's zero or not until we have the relevant data fetched from the remote.
It's not good for UX because with zero configured, the UI should have a value flashing issue (initially zero but after some time it gets updated as a correct value). It should be undefined.
In this specific case, this was suppressed by some other logic but on the vault dashboard page, we can see that issue clearly. Values transition from zero to correct values after some time (precisely once date has been fetched) there.